config/zfsnapphpdebugbar.config.php

```diff
@@ -3,7 +3,7 @@
 return array(
     'php-debug-bar' => array(
         'view' => array(
-            'custom-style-path' => 'zf-snap-php-debug-bar.css',
+            'custom-style-path' => 'zf-snap-php-debug-bar.min.css',
```

Comment:
Are we really need this? Debug bar it's only for development enviroment, so minify it's not necessary. We need to remember every time when source is change to rebuild it.